### PR TITLE
[IMP] fieldservice: Add default value on date start

### PR DESCRIPTION
**Pull request by Urvisha-OSI**
https://pm.opensourceintegrators.com/web#view_type=form&model=helpdesk.ticket&id=25757&active_id=25757&menu_id=

### CODE REVIEW

**Comment by max3903** [comment]
@Urvisha-OSI Can you fix the pre-commit errors?

File: fieldservice/models/fsm_order.py [hunk 1]
```diff
@@ -157,7 +157,7 @@ def _onchange_location_id_customer(self):
     # Planning
     person_id = fields.Many2one("fsm.person", string="Assigned To", index=True)
     person_phone = fields.Char(related="person_id.phone", string="Worker Phone")
-    scheduled_date_start = fields.Datetime(string="Scheduled Start (ETA)")
+    scheduled_date_start = fields.Datetime(string="Scheduled Start (ETA)", default=fields.Datetime.now())
```
Comment (dreispt) [inline fieldservice/models/fsm_order.py]: Wither use a lambda, or pass the function reference
```suggestion
    scheduled_date_start = fields.Datetime(string="Scheduled Start (ETA)", default=fields.Datetime.now)
```

**Review by dreispt — CHANGES_REQUESTED** [review]

**Comment by Urvisha-OSI** [comment]
> 



> @Urvisha-OSI Can you fix the pre-commit errors?

@max3903 The pre-commit error is solved.

**Review by dreispt — APPROVED** [review]

**Comment by dreispt** [comment]
Hmm, tests are failing
https://app.travis-ci.com/github/OCA/field-service/jobs/585145586#L2089

**Comment by max3903** [comment]
@Urvisha-OSI Can you fix the errors?

**Comment by Urvisha-OSI** [comment]
> @Urvisha-OSI Can you fix the errors?

@max3903 the error is fixed

**Comment by max3903** [comment]
@Urvisha-OSI https://app.travis-ci.com/github/OCA/field-service/jobs/586823193#L2156

**Review by max3903 — APPROVED** [review]

**Comment by Urvisha-OSI** [comment]
> @Urvisha-OSI https://app.travis-ci.com/github/OCA/field-service/jobs/586823193#L2156

@max3903 this is fails because of another module test cases fails there is not a issue with the fieldservice issue, so you can merge it to 14.0 it will not make any issue there because of the fieldservice module changes

**Comment by max3903** [comment]
@Urvisha-OSI The failing test is from the `fieldservice` module, not a different module:
https://app.travis-ci.com/github/OCA/field-service/jobs/586823193#L2156

**Comment by Urvisha-OSI** [comment]
> @Urvisha-OSI The failing test is from the `fieldservice` module, not a different module: https://app.travis-ci.com/github/OCA/field-service/jobs/586823193#L2156

@max3903 the travis fails because of fieldservice_calendar module, you can check it here: https://app.travis-ci.com/github/OCA/field-service/jobs/587325517
let me know if we want to fix the fieldservice_calendar error in this PR

**Comment by max3903** [comment]
@Urvisha-OSI Ok

**Comment by max3903** [comment]
@Urvisha-OSI Please fix the error in this PR.

**Comment by Urvisha-OSI** [comment]
> @Urvisha-OSI Please fix the error in this PR.

@max3903 all checks are passed now, you can merge it

**Review by bodedra — APPROVED** [review]
LG 👍

**Comment by dreispt** [comment]
Precommit config needs to be updated, by running `copier update`.

**Pull request merged**